lighthouse-core/report/html/renderer/snippet-renderer.js

```diff
@@ -129,6 +130,7 @@ class SnippetRenderer {
     // access the full element detail. Just being able to see the outer HTML isn't very useful.
     if (details.node && dom.isDevTools()) {
       const nodeContainer = dom.find('.lh-snippet__node', header);
+      // @ts-ignore: renderNode is protected.
```

Comment:
should we just remove `@protected` from `renderNode` since apparently it's not? (`@protected` must have been put on there a long time ago...)